resources/views/listing/partials/item.blade.php

```diff
@@ -16,7 +16,9 @@ class="mb-3 h-48 w-full rounded-t object-contain" :alt="item.name" :loading="con
                 />
                 <x-rapidez::no-image v-else class="mb-3 h-48 rounded-t" />
                 <div class="px-2">
-                    <div class="text-base font-medium">@{{ item.name }}</div>
+                    <div class="text-base font-medium">
+                        <x-rapidez::highlight attribute="name"/>
+                    </div>
```

Comment:
I think we don't need the wrapper? And if we're going with the component suggestion; we don't need the atttribute as `name` is the default.

```suggestion
                    <x-rapidez::highlight class="text-base font-medium"/>
```